### PR TITLE
Update MacOS sections in troubleshooting.rst

### PR DESCRIPTION
**Pull request by Dreamedc2015**
updated how to update section and catalina support section

### CODE REVIEW

**Comment by Dreamedc2015** [comment]
i checked it in a seperate viewer and it works

File: troubleshooting.rst [hunk 1]
```diff
@@ -51,22 +51,16 @@ No audio
...
+macOS X Catalina
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Currently, macOS Catalina does not support 32-bit applications at this time; you will not be able to use this launcher.
-There are some solutions, you can:
-
-- Downgrade to Mojave
-
-- Find a suitable Linux Distro (anything Ubuntu-based will be the easiest)
-
-File picking doesn't work or crashes
```
Comment (ChristopherHX) [inline troubleshooting.rst:69]: Why do you want to remove this title?

**Comment by ChristopherHX** [comment]
The check for updates button needs to be fully implemented first.
E.g. on linux you don't see a response, like on macOS.

You are absolutly right that the wiki needs to be updated to match my changes from this year.

File: troubleshooting.rst [hunk 2]
```diff
@@ -51,22 +51,16 @@ No audio
...
+if your launcher is too old or you cant find these buttons 
+in Ubuntu prebuilt packages do sudo apt-get update && sudo apt-get upgrade 
 
-- Mac OS - Redownload the package
+and on Mac OS - Redownload the package
 
-macOS X Catalina - :code:`Could not find the game launcher..`
+macOS X Catalina
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Currently, macOS Catalina does not support 32-bit applications at this time; you will not be able to use this launcher.
```
Comment (ChristopherHX) [inline troubleshooting.rst:62]: This mostly didn't changed with ng, but 
"you will not be able to use this launcher."
should be replaced by "you won't be able to run older Minecraft < 1.14.60.5"

File: troubleshooting.rst [hunk 3]
```diff
@@ -51,22 +51,16 @@ No audio
...
+to update the launcher on newer versions of the launcher Press the gear icon and then check for updates
 
-- Ubuntu prebuilt packages - :code:`sudo apt-get update && sudo apt-get upgrade`
+if your launcher is too old or you cant find these buttons 
+in Ubuntu prebuilt packages do sudo apt-get update && sudo apt-get upgrade 
 
-- Mac OS - Redownload the package
+and on Mac OS - Redownload the package
 
-macOS X Catalina - :code:`Could not find the game launcher..`
```
Comment (ChristopherHX) [inline troubleshooting.rst:60]: I replaced the `Could not find the game launcher..` message,
but still applies if someone find an old version and get this error

File: troubleshooting.rst (hunk 2 above)
Comment (Mhowser) [reply troubleshooting.rst:62]: I agree.

File: troubleshooting.rst (hunk 1 above)
Comment (Mhowser) [reply troubleshooting.rst:69]: I think this part should stay as they are still valid.

**Comment by Mhowser** [comment]
I'll merge it now and clean it up manually later, thanks!

**Pull request merged**